src/libtest/lib.rs

```diff
@@ -544,7 +552,11 @@ pub fn parse_opts(args: &[String]) -> Option<OptRes> {
         None
     };
 
-    let run_ignored = matches.opt_present("ignored");
+    let run_ignored = match (matches.opt_present("all"), matches.opt_present("ignored")) {
+        (true, _) => RunIgnored::Yes,
```

Comment:
I wonder if it makes sense to error here if we've been passed both; what do you think?

Reply:
Hmm, maybe. It's often better for humans but worse for automated tools.